cli.py

```diff
@@ -21,12 +21,14 @@ def cli():
 @cli.command()
 @click.argument("block_number", type=int)
 @click.option("--rpc", default=lambda: os.environ.get(RPC_URL_ENV, ""))
+@click.option("--geth/--no-geth", default=False)
 @coro
-async def inspect_block_command(block_number: int, rpc: str):
+async def inspect_block_command(block_number: int, rpc: str, geth: bool):
```

Comment:
Mentioned this in our channel a bit ago - I think it would be better to pass around `rpc_type` where it's an enum of `geth` or `parity` to make geth less "exceptional" and just make it one of our supported RPC types.

Then the code itself would read nicer as well
```
if rpc_type == RPCType.geth:
    return _fetch_block_geth(...)
elif rpc_type == RPCType.parity:
    return _fetch_block_parity(...)
...
```